### DISCUSSION TITLE
How to get GitHub Galaxy brain badge

### DISCUSSION THREAD

**Discussion by sanjay-kv** [recodehive/Stackoverflow-Analysis#46] [Announcements]
# 🏆 GitHub Achievements 🏆

<br>

<div align="center">
  <picture>
  
    <source media="(prefers-color-scheme: light)" srcset="https://user-images.githubusercontent.com/65187002/172940015-d9d072e7-c47d-4ddd-83f6-8e7717a721b8.png">
    <img src="https://user-images.githubusercontent.com/65187002/172940773-7ef23b63-3356-4634-9e52-34f2676e2854.png">
  </picture><br>
  <picture>
    <source media="(prefers-color-scheme: light)" srcset="https://user-images.githubusercontent.com/65187002/172941127-4061fac1-736b-4c24-b7ea-c210b3578cc5.png">
    <img width="50%" src="https://user-images.githubusercontent.com/65187002/172941149-31258408-bfc3-496a-8a58-e34794b21813.png">
  </picture>
</div>

<br>

# 🏅 Displaying Achievements 🏅

#### To see what it looks like visit my Profile: https://github.com/sanjay-kv :octocat: 

Why Achievement badge: The reason you are here is to do opensource, and it's not just about doing code, it's about helping the community, So answering somebodies questions is very important. GitHub provides you with a badge for the effort you took. 

Other people who visit your profile can see how helpful you are in opensource by analyzing the badge. So are you ready to help?
<br>

# 📃 Achievement List 📃
<br>

| Badge | Name | How to get | Needed amount | 
| :-: | :-: | :-: | :-: |
| ![Galaxy Brain](https://raw.githubusercontent.com/drknzz/GitHub-Achievements/main/Media/Badges/Galaxy-Brain/PNG/GalaxyBrain.png) | Galaxy Brain | Answered a discussion<br>(got an accepted answer) | <table>  <thead>  <tr>  <th>DEFAULT</th> <th>BRONZE</th>  <th>SILVER</th>  <th>GOLD</th>  </tr>  </thead>  <tbody>  <tr>  <td><img src="https://raw.githubusercontent.com/drknzz/GitHub-Achievements/main/Media/Badges/Galaxy-Brain/PNG/GalaxyBrain.png" width="60px"></td>  <td><img src="https://raw.githubusercontent.com/drknzz/GitHub-Achievements/main/Media/Badges/Galaxy-Brain/PNG/GalaxyBrain_Bronze.png" width="60px" align="center"></td>  <td><img src="https://raw.githubusercontent.com/drknzz/GitHub-Achievements/main/Media/Badges/Galaxy-Brain/PNG/GalaxyBrain_Silver.png" width="60px"></td>  <td><img src="https://raw.githubusercontent.com/drknzz/GitHub-Achievements/main/Media/Badges/Galaxy-Brain/PNG/GalaxyBrain_Gold.png" width="60px"></td>  </tr>  <tr>  <td align="center">2</td> <td align="center">8</td>  <td align="center">16</td>  <td align="center">32</td>  </tr>   </tbody>  </table>
<br>

**Comment by ujjwalshriv3** [comment]
You made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here Achievements will no longer be available in the Community.

Note that GitHub's Acceptable Use Policies prohibits coordinated or inauthentic activity like rapid questions and answers. If users create discussions of this nature, it may result in a temporary or indefinite block from the Community.

This change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

Thanks for understanding!

**Reply by sanjay-kv** [reply]
https://github.com/orgs/GSSoC24/discussions

to make it clear we are not letting anyone to do any violating policy things. 
As a organisation we tend to receive lot of queries through the email, DM. so these discussion is a great help to manage it. 

as you can see in the above link, because of this gamification, contributors also came forward to help. which resulted answering around 1000+ queries on 2 repos. I hope you understood the intentions.

**Reply by ujjwalshriv3** [reply]
Ok sir I understand thanks for explaining me

**Reply by Hassassin-10** [reply]
i need Galaxy Brain Badge Sir

**Reply by Kazuma275** [reply]
I need the badge too, thanks

**Reply by oceaneRoche** [reply]
I need the badge too

**Reply by iamAntimPal** [reply]
i neet the bedge too

**Reply by koushik369mondal** [reply]
i neet the bedge too

**Reply by phanduykhai05** [reply]
i need Galaxy Brain Badge Sir

**Reply by phanduykhai05** [reply]
I need the badge too, thanks

**Reply by jothirmoysarker** [reply]
me too!!

**Comment by RaviSingh733** [comment]
thanks

**Comment by DrDead0** [comment]
to earn the badge, you need to answer any 2 questions in any GitHub discussion, if the find the answer in your comment they whey will mark you answers as "marked as answer" upon that you will get that badge

**Reply by Hassassin-10** [reply]
Then mark me as correct

**Reply by DrDead0** [reply]
i cannot only maintainer and the owner of this discussion can mark you.!

**Comment by abubakaristiak** [comment]
How can I efficiently manage and contribute to multiple open-source repositories as a beginner in open-source development?

**Reply by Anonx31** [reply]
you can learn about the specific topic and can provide the acquired information into the discussion

**Comment by shoriful-dev** [comment]
Anyone here please mark me.

**Reply by DrDead0** [reply]
only collaborator can mark you and once been marked no other answer can get marked.!

**Comment by koushik369mondal** [comment]
how can I get Galaxy Brain badge on GitHub

**Reply by privatekaushik** [reply]
you have to be marked

**Comment by sandipanrakshit34** [comment]
hey please give me a star on this repository and help me to achieve the badge:

https://github.com/sandipanrakshit34/sandipanrakshit34

**Comment by indranil143** [comment]
https://github.com/indranil143/indranil143/edit/main/README.md

give it a star and fork it

**Comment by samueltexa** [comment]
Hey everyone! 👋
Just jumping into the discussions — looking forward to helping out where I can. If anyone has a topic or question that could use another perspective, feel free to tag me or drop a link. Always up for learning and sharing what I know. 🙌

**Comment by shoriful-dev** [comment]
i need the Galaxy Brain badge

**Comment by Lyynov** [comment]
i need the Galaxy Brain badge

**Comment by saif-pain** [comment]
Done

**Comment by A-Hemeda** [comment]


**Reply by 00-Source** [reply]
Hey Ahmed, welcome to the open-source community! 👋

You're spot on—contributing with the intention to help rather than just earn recognition is the right mindset. The Galaxy Brain badge is definitely a nice bonus, but the real value comes from building relationships and improving your skills through collaboration.

In my case, the accepted answers that earned the badge usually had a few things in common:

Clarity: I made sure the explanation was easy to follow, even for beginners.

Practical examples: Sharing a working code snippet or linking to relevant documentation added a lot of value.

Context awareness: Tailoring the answer to the specific problem, rather than giving a generic response, helped it stand out.

Also, asking clarifying questions before posting an answer often led to better responses, and sometimes that engagement itself was appreciated enough to get marked as accepted.

Curious to hear from others too—what worked for you?

**Comment by xDimon** [comment]
Please, look at my pet-project: https://github.com/xDimon/soralog

If it seems useful for community, start it please.

**Comment by phanduykhai05** [comment]
hey please give me a star on this repository and help me to achieve the badge:
https://github.com/phanduykhai05/phanduykhai05

**Comment by Abhi19990628** [comment]
Thanks for the clarification and the announcement.

I understand the reasoning behind disabling Achievements in GitHub Community Discussions. Encouraging authentic participation and preventing badge farming helps maintain the quality of discussions for everyone.

It's also good to know that contributors can still earn badges through meaningful contributions in other GitHub projects and communities.

Appreciate the transparency and the efforts to keep the community healthy and valuable for developers.

**Comment by Abhi19990628** [comment]
This makes sense. While achievements were fun, maintaining authentic discussions is more important for the long-term health of the community.

I appreciate GitHub's efforts to encourage meaningful contributions over badge hunting. Looking forward to continuing to learn and contribute across the GitHub ecosystem.

**Comment by phanduykhai05** [comment]
https://github.com/phanduykhai05
i need the Galaxy Brain badge
